packages/playground/_stories/main/Label/Label.stories.ts

```diff
@@ -29,45 +29,85 @@ const Template: UI5StoryArgs<Label, StoryArgsSlots> = (args) => {
 	?required="${ifDefined(args.required)}"
 	?show-colon="${ifDefined(args.showColon)}"
 	wrapping-type="${ifDefined(args.wrappingType)}"
+	class="${ifDefined(args.className)}"
 >
 	${unsafeHTML(args.default)}
 </ui5-label>`;
 };
 
+const addInput = (id: string) => {
+	return (story: () => unknown) => html`
+${story()}
+<ui5-input id="${id}"></ui5-input>`;	
+};
```

Comment:
```ts
const addInput = (id: string): Decorator => {
	return (story, { args }) => {
		return html`
${story({ args: { ...args, for: id } })}
<ui5-input id="${id}"></ui5-input>`;
	};
};
```
the decorator import is from `import type { Decorator } from "@storybook/web-components"`